doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -540,68 +511,52 @@ use one or several databases specific to your development.
 
 .. _removehints:
 
-.. cmd:: Remove Hints {+ @term} : {+ @ident}
+.. cmd:: Remove Hints {+ @qualid } {? : {+ @ident } }
 
    This command removes the hints associated to terms :n:`{+ @term}` in databases
    :n:`{+ @ident}`.
```

Comment:
Description should be fixed. You can just say that it removes the hints associated to the `qualid` (and maybe add a note that hints introduced with `Hint Extern` currently cannot be removed).

Reply:
Would making the hints local be a partial workaround for `Hint Extern`?  If so, I'll mention.

BTW, IMHO it should be `associate with`, not `associate to`.

Reply:
The best workaround is to make the hints non-global (e.g. using the `export` attribute) and carefully select which modules you import.